modules/custom/core/govcms8_modifiers/config/install/field.storage.paragraph.field_mod_image.yml

```diff
@@ -0,0 +1,20 @@
+uuid: 75cd7c3a-165a-4ab6-972e-6bf9c3779a79
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+    - paragraphs
+id: paragraph.field_mod_image
```

Comment:
Can we make the Image Background Modifier use the same media browser as the featured image field for consistency?

Reply:
Fixed in e2cb3b8cb13a28334ef135040adb76017c0a2098